.kitchen.yml

```diff
@@ -82,3 +82,22 @@ suites:
     encrypted_data_bag_secret_key_path: "test/integration/forwarder/encrypted_data_bag_secret"
     run_list:
       - recipe[elkstack::forwarder]
+
+<% if ENV['RACKSPACE_MONITORING_USERNAME'] && ENV['RACKSPACE_MONITORING_API_KEY'] %>
```

Comment:
The goal here is to only test monitoring if we have explicitly defined 2 ENV variables, I could use `RACKSPACE_API_KEY` and `RACKSPACE_USERNAME` which is supported by `knife rackspace`, but I was not sure we wanted to do this by default.